pandera/io.py

```diff
@@ -404,3 +405,196 @@ def to_script(dataframe_schema, path_or_buf=None):
 
     with Path(path_or_buf).open("w") as f:
         f.write(formatted_script)
+
+
+class FrictionlessFieldParser:
+    """Parses frictionless data schema field specifications so we can convert
+    them to an equivalent pandera Column schema.
+
+    For this implementation, we are using field names, constraints and types
+    but leaving other frictionless parameters out (e.g. foreign keys, type
```

Comment:
we can go back and add this feature when #331 is done

Reply:
in anticipation of this, I've standardised all properties of `FrictionlessFieldParser` to reflect the properties we need for a pandera Column object - so when #331 is complete it'll be clearer where to add the new properties in, too.